data/json/mapgen/homeimprovement_superstore.json

```diff
@@ -222,6 +222,8 @@
         "c": "t_floor",
         "r": "t_floor",
         "s": "t_floor",
+		"O": "t_floor",
```

Comment:
Tabs instead of spaces.

Reply:
Sorry about that - missed one. This is not my usual computer. Will have the corrected push in a minute.